### DISCUSSION TITLE
sudo anonsurf myip

### DISCUSSION THREAD

**Discussion by WhiteHatCatx** [community/community#113700] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Does anyone know why I would be getting the following output after running "sudo anonsurf myip"?

─$ sudo anonsurf myip

My ip is:

<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>403 Forbidden</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Forbidden</h1>
<h2>Your client does not have permission to get URL <code>/</code> from this server.</h2>
<h2></h2>
</body></html>

**Comment by vickkie** [comment]
### Hello @WhiteHatCatx 

does that happen whjen connected under tor network or even when not connected?

### This might be due to several reasons
- The server anonsurf is trying to access might have been configured to deny requests from certain IP addresses or ranges, including those that might be associated with Tor exit nodes. This is a common security measure to prevent abuse.
- IP Blocking: Your IP address might be blocked by the server for various reasons, such as previous suspicious activity.
Network Configuration: There might be a network configuration issue on your end or with your ISP that's preventing anonsurf from making the request successfully.
- Your anonsurf Configuration: It's also possible that anonsurf itself is configured in a way that's causing this issue. For example, it might be using a server that's no longer operational or has changed its access policies.


You might want to update anonysurf to the latest using apt if you using linux based system
